doc/release/1.2.0-notes.rst

```diff
@@ -2,32 +2,91 @@
+
+`differential_evolution` now has the `updating` and `workers` keywords. The first chooses between continuous updating of the best solution vector (the default), or once per generation. Continuous updating can lead to faster convergence. The `workers` keyword accepts an `int` or map-like callable, and parallelises the solver (having the side effect of updating once per generation). Supplying an `int` evaluates the trial solutions in N parallel parts. Supplying a map-like callable allows other parallelisation approaches (such as `mpi4py`, or `joblib`) to be used.
+
+`dual_annealing` is a new method which uses two annealing processes to accelerate the convergence towards the global minimum of an objective mathematical function. The first annealing process controls the stochastic Markov chain searching and the second annealing process controls the deterministic minimization.  
+So, dual annealing is a hybrid method that takes advantages of stochastic and local deterministic searching in an efficient way.
+
+`shgo` (simplicial homology global optimization) is a powerful new general purpose global optimization (GO) algorithm appropriate for solving black box and derivative free optimization (DFO) problems. The algorithm is proven to convergence to the global solution in finite time and shows [highly competitive performance compared to cutting edge commercial and open source algorithms available in literature](https://stefan-endres.github.io/files/shgo_jogo.pdf). The convergence holds for non-linear inequality and equality constraints. In addition to returning a global minimum, the algorithm also returns any other global and local minima found after every iteration. This makes it useful for exploring the solutions in a domain.
+
+`scipy.signal` improvements
+---------------------------
+
+Digital filter design functions now include a parameter to specify the sampling rate.  Previously, digital filters could only be specified using normalized frequency, but different functions used different scales (e.g. 0 to 1 for `butter` vs 0 to π for `freqz`), leading to errors and confusion.  With the `fs` parameter, ordinary frequencies can now be entered directly into functions, with the normalization handled internally.
+
+`find_peaks` and related functions no longer raise an exception if the properties of a peak have unexpected values (e.g. a prominence of 0). A `PeakPropertyWarning` is given instead.
+
+The new keyword argument `plateau_size` was added to `find_peaks`. `plateau_size` may be used to select peaks based on the length of the flat top of a peak.
+
+`scipy.sparse` improvements
+---------------------------
+
+The `scipy.sparse.bsr_matrix.tocsr` method was implemented directly instead of converting via COO format, and the `scipy.sparse.bsr_matrix.tocsc` method was also routed via CSR conversion instead of COO. The efficiency of both conversions was improved.
```

Comment:
(ESL here). Does past tense (smth was implemented) refer to previous releases or to what was done for this one?